specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-06-01/containerregistry_build.json

```diff
@@ -0,0 +1,2578 @@
+      "properties": {
+        "id": {
+          "description": "The unique ID of the trigger.",
+          "type": "string"
+        },
+        "timestamp": {
+          "format": "date-time",
+          "description": "The timestamp when the image update happened.",
+          "type": "string"
+        },
+        "images": {
+          "description": "The list of image updates that caused the build.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ImageDescriptor"
+          }
+        }
+      }
+    },
+    "SourceTriggerDescriptor": {
+      "description": "The source trigger that caused a run.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The unique ID of the trigger.",
+          "type": "string"
+        },
+        "eventType": {
+          "description": "The event type of the trigger.",
+          "type": "string"
+        },
+        "commitId": {
+          "description": "The unique ID that identifies a commit.",
+          "type": "string"
+        },
+        "pullRequestId": {
+          "description": "The unique ID that identifies pull request.",
+          "type": "string"
+        },
+        "repositoryUrl": {
+          "description": "The repository URL.",
+          "type": "string"
+        },
+        "branchName": {
+          "description": "The branch name in the repository.",
+          "type": "string"
+        },
+        "providerType": {
+          "description": "The source control provider type.",
+          "type": "string"
+        }
+      }
+    },
+    "PlatformProperties": {
+      "description": "The platform properties against which the run has to happen.",
+      "required": [
+        "os"
+      ],
+      "type": "object",
+      "properties": {
+        "os": {
+          "description": "The operating system type required for the run.",
+          "enum": [
+            "Windows",
+            "Linux"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "OS",
+            "modelAsString": true
+          }
+        },
+        "architecture": {
+          "description": "The OS architecture.",
+          "enum": [
+            "amd64",
+            "x86",
+            "arm"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Architecture",
+            "modelAsString": true
+          }
+        },
+        "variant": {
+          "description": "Variant of the CPU.",
+          "enum": [
+            "v6",
+            "v7",
+            "v8"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Variant",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "AgentProperties": {
+      "description": "The properties that determine the run agent configuration.",
+      "type": "object",
+      "properties": {
+        "cpu": {
+          "format": "int32",
```

Comment:
Can we do min and max here?

Reply:
@northtyphoon do we have a min and max here?

Reply:
@kpajdzik I dont think we have a min and max here.
cc @northtyphoon